src/tools/skinManagerGui/CMakeLists.txt

```diff
@@ -36,6 +36,9 @@ add_executable(skinManagerGui WIN32    ${skinManagerGui_SRCS}
                                        ${skinManagerGui_HDRS}
                                        ${skinManagerGui_QRC_GEN_SRCS}
                                        ${skinManagerGui_UI_GEN_SRCS})
+if(WIN32)
+  target_compile_definitions(skinManagerGui PUBLIC _HAS_STD_BYTE=0)
+endif()
```

Comment:
Apparently, there's a [native problem with some Windows SDK and `std::byte` defined in C++17][des].
This problem impacts the compilation of some `skinManagerGui` auto-generated files.

Here's the proposed [solution][sol].

We also need to be careful as in the future [`_HAS_STD_BYTE` will get removed][rem].

[des]: https://developercommunity.visualstudio.com/t/error-c2872-byte-ambiguous-symbol/93889
[sol]: https://developercommunity.visualstudio.com/t/error-c2872-byte-ambiguous-symbol/93889#T-N859630
[rem]: https://github.com/microsoft/STL/issues/20

cc @Nicogene @traversaro @vtikha

Reply:
> [rem]: https://github.com/microsoft/STL/issues/20

I guess there was a typo and the correct issues is https://github.com/microsoft/STL/issues/204 ?

Reply:
> Can we open an issue on how to fix this in a more permanent way? Perhaps just avoiding a windows include or a `using namespaces std` somewhere in skinManagerGui could help in this case.

Unfortunately, the failure is triggered within files that are generated by Qt, which in turn do include the "problematic" Windows SDK resources.

After a quick F2F alignment, we've decided to go this way.